### DISCUSSION TITLE
v1.9.3

### DISCUSSION THREAD

**Discussion by ysfscream** [emqx/MQTTX#1302] [General]
## What's Changed
* feat(gtm): add google tag manager by @Red-Asuka in https://github.com/emqx/MQTTX/pull/1272
* fix(cli): fix typo by @Red-Asuka in https://github.com/emqx/MQTTX/pull/1274
* docs(readme): update readme by @ysfscream in https://github.com/emqx/MQTTX/pull/1275
* refactor(logo): update mqttx logo by @ysfscream in https://github.com/emqx/MQTTX/pull/1277
* refactor: fix shields icon name to MQTTX by @JimMoen in https://github.com/emqx/MQTTX/pull/1278
* Revert "chore(deps): bump sqlite3 from 5.0.4 to 5.1.5" by @ysfscream in https://github.com/emqx/MQTTX/pull/1282
* feat(simulate): add simulate to cli by @wivwiv in https://github.com/emqx/MQTTX/pull/1280
* fix(connections): set auto reconnect to true by default by @ysfscream in https://github.com/emqx/MQTTX/pull/1284
* docs(readme): update readme by @ysfscream in https://github.com/emqx/MQTTX/pull/1285
* fix(connections): disable edit in connecting status by @ysfscream in https://github.com/emqx/MQTTX/pull/1286
* refactor(scrips-example): move the scripts-example to top & add iot data scenarios scripts by @ysfscream in https://github.com/emqx/MQTTX/pull/1287
* fix(cli): add js file validation by @ysfscream in https://github.com/emqx/MQTTX/pull/1288
* docs(cli): update readme of simulate by @ysfscream in https://github.com/emqx/MQTTX/pull/1290
* chore(cli): update .npmignore file by @Red-Asuka in https://github.com/emqx/MQTTX/pull/1291
* chore(pkg): upgrade version 1.9.3 by @ysfscream in https://github.com/emqx/MQTTX/pull/1293
* feat(cli): add ls command by @ysfscream in https://github.com/emqx/MQTTX/pull/1292
* docs(cli): update ls command by @ysfscream in https://github.com/emqx/MQTTX/pull/1294
* feat(simulate): add industrial energy monitoring scenario script by @wivwiv in https://github.com/emqx/MQTTX/pull/1296
* chore(dockerfile): use alpine images to reduce the size of Docker images by @wivwiv in https://github.com/emqx/MQTTX/pull/1298
* feat(cli): add smart home scenario by @ysfscream in https://github.com/emqx/MQTTX/pull/1299
* feat(cli): add weather scenario by @ysfscream in https://github.com/emqx/MQTTX/pull/1300
* chore(cli): reduce the docker image size by @ysfscream in https://github.com/emqx/MQTTX/pull/1301

## New Contributors
* @JimMoen made their first contribution in https://github.com/emqx/MQTTX/pull/1278
* @wivwiv made their first contribution in https://github.com/emqx/MQTTX/pull/1280

**Full Changelog**: https://github.com/emqx/MQTTX/compare/v1.9.2...v1.9.3

<hr /><em>This discussion was created from the release <a href='https://github.com/emqx/MQTTX/releases/tag/v1.9.3'>v1.9.3</a>.</em>

**Comment by ves011** [comment]
cannot install it because it complains mqttx is running.
I cannot see any mqttx... process in task manager
Windows 10 pro 64 bits, build 19044.3086
